src/sss_client/ssh/sss_ssh_knownhostsproxy.c

```diff
@@ -18,7 +18,27 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include <stdio.h>
+#include <stdlib.h>
+
+#include "config.h"
+#include "util/util.h"
+
+#ifndef BUILD_SSH_KNOWN_HOSTS_PROXY
+
+int main(void)
+{
+    ERROR("\n"
+          "******************************************************************************\n"
+          "Your system is configured to use the sss_ssh_knownhostsproxy tool which has been removed.\n"
```

Comment:
Hi,

I would wrap this line to follow the 79 character limit defined by the surrounding `***` lines.

bye,
Sumit

Reply:
> I would wrap this line to follow the 79 character limit defined by the surrounding `***` lines.

Done

Reply:
Thanks.